### PR TITLE
Prop to make beacon inactive (i.e. not visible)

### PR DESCRIPTION
**Pull request by matthewgertner**

### CODE REVIEW

File: assets/style.styl [hunk 1]
```diff
@@ -162,16 +162,6 @@ tour-tooltip
...
-  tour-settings
-    display block
-    line-height 1
-    white-space nowrap
-    color #811b34
-    margin-top 15px
-    text-transform uppercase
-    font-size smaller
-    cursor pointer
-
```
Comment (matthewgertner) [inline assets/style.styl:174]: Not entirely sure how this got into `develop`. It'll be in a future PR for the "don't show hints" feature.

File: src/Beacon.jsx [hunk 2]
```diff
@@ -14,16 +14,17 @@ const TOOLTIP_TOLERANCE = 50;
 
 function deepCloneNode(node, parent, fn) {
   const clone = fn(node, parent);
-  const childNodes = [...node.childNodes];
+  const childNodes = Array.prototype.slice.call(node.childNodes);
```
Comment (matthewgertner) [inline src/Beacon.jsx:17]: Small unrelated change to fix an iOS 8 issue.

Comment (tomkis) [inline src/Beacon.jsx:17]: Can you please elaborate? Looks like a magic to me and definitely worth adding a comment here.

File: src/Beacon.jsx [hunk 3]
```diff
@@ -228,7 +230,9 @@ export class Beacon extends Component {
     return (this.state.tooltipAttachment &&
       <TetherComponent {...tetherProps}>
         {this.props.children}
-        <tour-tooltip class={tooltipClass} ref="tooltip" dangerouslySetInnerHTML={{__html: this.props.tooltipText}} />
+        <tour-tooltip class={tooltipClass} ref="tooltip">
```
Comment (tomkis) [inline src/Beacon.jsx:233]: 👍

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-beacon",
-  "version": "0.8.3",
+  "version": "0.8.5",
```
Comment (tomkis) [inline package.json:3]: So where is version 0.8.4 ?

File: src/Beacon.jsx [hunk 5]
```diff
@@ -176,7 +178,7 @@ export class Beacon extends Component {
   }
 
   renderBeacon(persistent) {
-    if (persistent && !this.state.hash) {
+    if (this.state.inactive || (persistent && !this.state.hash)) {
       // We need to wait until the hash is set before we render.
```
Comment (tomkis) [inline src/Beacon.jsx:182]: This comment does not reflect the intention anymore

**Comment by tomkis** [comment]
LGTM
